### PR TITLE
My first chord

### PR DESCRIPTION
**Pull request by TheWeaVer**
**Motivation**
* Create first chord playground, dive into lia project

**Changes**
* Build my own chord playground

### CODE REVIEW

**Review by zozond — APPROVED** [review]
LGTM

File: hephaistus/main-theme/distribution-system/chord/playground/TheWeaVer/my-chord/src/main/java/com/lia/hephaistus/chord/theweaver/DefaultChordNode.java [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * The 128 size chord node(m = 7).
+ * It does not generate any key, but throws exception if the given key is lager than 128.
+ */
+public class DefaultChordNode implements ChordNode {
```
Comment (nullist0) [inline hephaistus/main-theme/distribution-system/chord/playground/TheWeaVer/my-chord/src/main/java/com/lia/hephaistus/chord/theweaver/DefaultChordNode.java:12]: [nit] If there is a functions to be implemented in later PR, then I think it is better to add some comment or some code like `throw Exception("Not yet implemented")`.

**Review by nullist0 — APPROVED** [review]
LGTM with nit comments

File: hephaistus/main-theme/distribution-system/chord/playground/TheWeaVer/my-chord/src/main/java/com/lia/hephaistus/chord/theweaver/DefaultChordNode.java (hunk 1 above)
Comment (TheWeaVer) [reply hephaistus/main-theme/distribution-system/chord/playground/TheWeaVer/my-chord/src/main/java/com/lia/hephaistus/chord/theweaver/DefaultChordNode.java:12]: Thank you, I apply it on my IDE
image (1)

**Pull request merged**